File: 3-Ruby_Programming/rubymisc/euler1.rb
puts "We will calculate the sum of multiples of three and five."
puts "What would you like the maximum number set to?"
input = gets.to_f
@fives = 0.0
@fiveround = 0.0
@threes = 0.0
@threeround = 0.0
@fifteens = 0.0
@fifteenround = 0.0
def sum_of_multiples(input)
	val = input - 1
	@fiveround = (val - (val % 5))
	@fives = @fiveround / 5
	@threeround = (val - (val % 3))
	@threes = @threeround / 3
	@fifteenround = (val - (val % 15))
	@fifteens = @fifteenround / 15

	@sumfives = (@fives) * ((@fiveround + 5) / 2)
	@sumthrees = (@threes) * ((@threeround + 3) / 2)
	@sumfifteens = (@fifteens) * ((@fifteenround + 15) /2)

	@sum = (@sumfives + @sumthrees - @sumfifteens).to_i
end

puts sum_of_multiples(input)

File: 2-Web_Development_101/euler/euler1.js
var isMultiple = function(number){
if ((number % 3 === 0) || (number % 5 === 0)) {
    return number;
}
    else {
        return 0;
    }
};

var sumOfMultiples = function(){
    var input = document.getElementById("problem_1_input").value;
    var answer = document.getElementById("answer");
    var sum = 0
    for(var i = 1; i < input; i++){
        sum += isMultiple(i);
    }
    answer.innerHTML = sum;
};
